news/2018-06-17-project-loved-week-of-june-17th.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Project Loved: Week of June 17th"
+date: 2018-06-17 20:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back with a fresh new set of candidates for the Loved category! Come check them out, and vote for your favourites!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! This week's round of voting showcases some incredibly difficult beatmaps that are sure to give you a challenge. Across the four gamemodes, we've selected 28 maps, and now it's your turn to decide whether or not these will enter the Loved category. If a map has at least 75% positive votes for osu! or osu!taiko, or 80% for osu!catch or osu!mania, it'll make its way into the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=8)!
+
+Next week, these thresholds will be changing. osu! and osu!taiko will be increasing to 80%, following feedback that we've gotten from you! Both of these modes will also be cutting their candidate count down to five—[Toy](https://osu.ppy.sh/users/2757689) believes that this will ensure that all five candidates are strong picks, and it'll make sure that there's no shortage of awesome maps in the future.
```

Comment:
spaces before and after hyphen?

Reply:
it's an em dash, not a hyphen. I don't think you're supposed to put spaces around them